map-style-change/index.html

```diff
@@ -44,26 +44,28 @@
       <label for="dark">Dark</label>
     </div>
     <script src="https://unpkg.com/maplibre-gl"></script>
-    <script src="https://unpkg.com/amazon-location-helpers"></script>
+    <script src="AUTH URL TBD"></script>
```

Comment:
I believe we can derive the URL from the NPM package name, right? So it will be ``https://www.unpkg.com/@aws/amazon-location-authentication-helper``?